royalties-registry/contracts/providers/RoyaltiesProviderArtBlocks.sol

```diff
@@ -0,0 +1,68 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity >=0.6.9 <0.8.0;
+pragma abicoder v2;
+
+import "@rarible/royalties/contracts/IRoyaltiesProvider.sol";
+import "./RoyaltyArtBlocks.sol";
+import "../lib/BpLibrary.sol";
+import "@openzeppelin/contracts-upgradeable/access/OwnableUpgradeable.sol";
+
+contract RoyaltiesProviderArtBlocks is IRoyaltiesProvider, OwnableUpgradeable {
+    using SafeMathUpgradeable for uint;
+    using BpLibrary for uint;
+
+    uint96 artblocksPercentage;
+
+    event ArtblocksPercentageChanged(address _who, uint96 _old, uint96 _new);
+
+    function __RoyaltiesProviderArtBlocks_init() external initializer {
+        __Ownable_init();
```

Comment:
Let's use not upgradeable version of contracts here. no plan to upgrade this, then it's easier to use regular Ownable